docs/Examples/RxSwiftErrorHandling.md

```diff
@@ -0,0 +1,49 @@
+Handle errors with RxSwift
+==========================
+
+Sometimes you don't want to add error handling code manually to every
+request. Here is extension for `Observable` to add simple error handling:
+
+```swift
+extension Observable {
+    func showErrorHUD() -> Observable<Element> {
+        return self.doOn { event in
+            switch event {
+            case .Error(let e):
+				// Unwrap underlying error
```

Comment:
Could you align the left margin with the line below, please? And also the comment below as well.